src/application.ts

```diff
@@ -0,0 +1,55 @@
+import {Firestore} from '@google-cloud/firestore';
+import {GitHub} from './utils/github';
+
+export type ApplicationSecrets = {
+  GITHUB_CLIENT_ID: string; GITHUB_CLIENT_SECRET: string;
+  PUBLIC_VAPID_KEY: string;
+  PRIVATE_VAPID_KEY: string;
+};
+
+type Init = {
+  secrets: ApplicationSecrets|null; firestore: Firestore | null;
+  github: GitHub | null;
+};
+
+class Application {
```

Comment:
Does this need to be a class? If we're doing globals, maybe we could just have a module called "environment" that exports a bunch of globals. I find the name Application a little confusing, since it overlaps conceptually with the server class itself.